features/login.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Login page
+  As an User
+  In order to access my account
+  I need to be able to login
+
+  Background:
+    Given I am on the "login" page
+    And the following users exist
+    | email                | password    |
+    | orphanage@random.com | my_password |
+
+  Scenario: Accessing the login page
+    Given I am on the "home" page
+    And I click "Login"
+    Then I should be on the "login" page
+
+  Scenario: Login User
+    And I fill in "Email" with "orphanage@random.com"
```

Comment:
I think rather than starting with `And` you could use `When`. 

Then you have something like:
```gherkin
Scenario: Login User
  When I fill in "Email" with "orphanage@random.com"
  And I fill in "Password" with "my_password"
  ...
```